js/bookmarkConverter.js

```diff
@@ -81,6 +84,33 @@ const bookmarkConverter = {
       })
         .catch(reject)
     })
+  },
+  initialize: function () {
+    // how often to create a new backup file
+    const interval = (3 * 24 * 60 * 60 * 1000)
```

Comment:
I don't think the exact timing of the backup matters too much; the chances of someone creating a lot of bookmarks in 3 days and then not being able to remember them well enough to recover them seems kind of low. The backup is mostly to prevent months or years of bookmarks getting lost.